src/components/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import style from './ToggleSwitch.st.css';
+import {
+  ToggleSwitch as CoreToggleSwitch,
+  ToggleSwitchProps as CoreToggleSwitchProps,
+} from 'wix-ui-core/toggle-switch';
+
+import {
+  TPAComponentsContext,
+  TPAComponentsConsumer,
+} from '../TPAComponentsConfig';
+
+export interface ToggleSwitchProps extends CoreToggleSwitchProps {}
+
+export class ToggleSwitch extends React.Component<ToggleSwitchProps> {
+  static contextType = TPAComponentsContext;
+  static displayName = 'ToggleSwitch';
+  static defaultProps = { ...CoreToggleSwitch.defaultProps };
+
+  render() {
+    const { checked, disabled, ...rest } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile }) => (
```

Comment:
We're not using the `mobile` state, so we can remove `TPAComponsntsConsumer` and the `mobile` render prop